app/__tests__/CardService.test.js

```diff
@@ -0,0 +1,12 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import CardService from "../components/services/CardService";
+
+test("should find a button", () => {
```

Comment:
this test does not make sense

you are asserting the test creates a button, not that a button is created in a component

Reply:
correction done.